src/dbus.c

```diff
@@ -1117,6 +1117,63 @@ void signal_notification_closed(struct notification *n, enum reason reason)
         }
 }
 
+void signal_notification_removed(struct notification *n, enum reason reason)
```

Comment:
i am not sure this signal is necessary. because we already have the one for closed and for history removed.

to my understanding remove_current does a close + remove_history